Source/WebCore/platform/graphics/GeometryUtilities.h

```diff
@@ -98,6 +98,8 @@ std::array<FloatPoint, 4> verticesForBox(const FloatRect&, const FloatPoint);
 float toPositiveAngle(float angle);
 float toRelatedAcuteAngle(float angle);
 
+float normalizeAngle(float radians);
```

Comment:
It has been a trouble for me to recognize the units of the angle from the name of the angle functions. For example the two functions above `toPositiveAngle()` and `toRelatedAcuteAngle()` take angles in degree but this new function takes angle in radians. Can we choose a more descriptive name which can reveal the units of the angle, e.g  `normalizeAngleInRadians()` or something else?